.gitignore

```diff
@@ -1,5 +1,6 @@
 *~
 Manifest.toml
+desktop.ini
```

Comment:
is this needed?

Reply:
This ignores Windows' search breadcrumb files, not a big deal.